src/shoobx/wfmc/interfaces.py

```diff
@@ -378,9 +378,12 @@ class IParameterDefinition(interface.Interface):
 
     input = interface.Attribute("Is this an input parameter?")
 
+    skipInputEval = interface.Attribute("Should we skip evaluating the input parameter?")
```

Comment:
How do you set this to True in practice (in xpdl)? Does it require to change xpdl spec to be settable?

Reply:
Hmm, was planning on using MODE=LITERALIN in the Xpdl (current values are "IN/OUT/INOUT"):

`<xpdl:FormalParameter Id="send_email" IsArray="false" Mode="INOUT" Name="Send Email">`

Our XPDL editor loads the new MODE='LITERALIN' fine, but on save rewrites it to MODE='IN'. Looking into it.